packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -52,6 +53,7 @@ enum _ScaffoldSlot {
   snackBar,
   floatingActionButton,
   drawer,
+  statusBar,
```

Comment:
i don't understand why this is a new thing. Shouldn't the appbar be the thing here?